exercises/raindrops/instructions.md

```diff
@@ -0,0 +1,23 @@
+# Instructions
+
+Your task is to convert a number into its corresponding raindrop sounds.
+
+If a given number:
+
+- is divisible by 3, add 'Pling' to the result.
+- is divisible by 5, add 'Plang' to the result.
+- is divisible by 7, add 'Plong' to the result.
```

Comment:
I think most languages use double quotes for strings, so maybe we use them there too?

```suggestion
- is divisible by 3, add "Pling" to the result.
- is divisible by 5, add "Plang" to the result.
- is divisible by 7, add "Plong" to the result.
```